contentScript.js

```diff
@@ -118,14 +118,14 @@
       itemList.appendChild(item);
       item.toolbar = toolbar;
       item.answer = answers[i].description;
-      item.addEventListener('click', insertAnswer);
+      $(item).on('selectmenu:selected', insertAnswer);
```

Comment:
Alright, I'll just revert this part of the change for now. I think the other class name changes should be safe.